llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -352,6 +352,16 @@ void FunctionImporter::ImportMapTy::maybeAddDeclaration(
   ImportMap[FromModule].try_emplace(GUID, GlobalValueSummary::Declaration);
 }
 
+SmallVector<StringRef, 0>
+FunctionImporter::ImportMapTy::getSourceModules() const {
+  SetVector<StringRef> ModuleSet;
```

Comment:
Why SetVector? The modules should already be deduplicated as map keys. Why not just push_back into a vector?

Also, can you mention the change from std::set for sorting to sorting of a vector in the comments - this should be more efficient I believe.

Reply:
A good point!  I've switched to `SmallVector` in the latest iteration.  Thanks!